apps/frontend/core-dapp/src/features/trade/close-trade/ClosePositionButton.tsx

```diff
@@ -32,10 +32,10 @@ const ClosePositionButton: React.FC = () => {
     if (
       closePositionValueBN !== undefined &&
       closePositionValueBN.gt(0) &&
-      selectedPosition.totalValueBN !== undefined
+      selectedPosition.totalValueInETHBN !== undefined
```

Comment:
nit: use camel case for Eth for readability
```suggestion
      selectedPosition.totalValueInEthBN !== undefined
```

Reply:
Same, these were introduced in previous PRs so I didn't want to touch existing code